render/mapping.go

```diff
@@ -49,6 +32,88 @@ type MapFunc func(report.NodeMetadata) (RenderableNode, bool)
 // node IDs prior to mapping.
 type PseudoFunc func(srcNodeID string, srcNode RenderableNode, dstNodeID string) (RenderableNode, bool)
 
+// MapFunc is anything which can take an arbitrary RenderableNode and
+// return another RenderableNode.
+type MapFunc func(RenderableNode) (RenderableNode, bool)
```

Comment:
As with LeafMapFunc, allows you to not emit node (when false).  I'll extend the comment.